tools/travis/build.sh

```diff
@@ -40,6 +40,8 @@ $ANSIBLE_CMD openwhisk.yml
 
 cd $ROOTDIR
 cat whisk.properties
+
+tail -f /tmp/wsklogs/invoker0/invoker0_logs.log /tmp/wsklogs/invoker1/invoker1_logs.log /tmp/wsklogs/controller0/controller0_logs.log /tmp/wsklogs/controller1/controller1_logs.log | grep ERROR &
```

Comment:
maybe make it a new PR and be more generic to cover more files **/.log and grep for ERROR after the logs.yml collected everything.

Reply:
great suggestion @csantanapr .

Reply:
Doing this will I think overflow travis' log limit (at least that was the case when I tried this the last time).